test/test_ops.py

```diff
@@ -422,30 +425,35 @@ def out_fn(output):
                             return sample.output_process_fn_grad(output)
                         return output
 
+                    def get_sample():
+                        return clone_input_helper(sample.input) if op.name[-1] == '_' else sample.input
```

Comment:
The above comment suggests this test does not handle inplace operations properly at all, and instead of checking an operator's name it would be preferable to check if the current variant was the operator's inplace variant (by comparing them directly)